packages/slider/test/__snapshots__/index.js.snap

```diff
@@ -1,8 +1,68 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Slider renders 1`] = `
+.c0 {
```

Comment:
I've been seeing this snapshot change – might be something weird going on with Lerna bootstrap